modules/fee-market/src/s2s/payment.rs

```diff
@@ -41,19 +41,19 @@ pub struct FeeMarketPayment<T, I, Currency> {
 	_phantom: sp_std::marker::PhantomData<(T, I, Currency)>,
 }
 
-impl<T, I, Currency> MessageDeliveryAndDispatchPayment<T::Origin, T::AccountId, RingBalance<T>>
+impl<T, I, Currency> MessageDeliveryAndDispatchPayment<T::Origin, T::AccountId, RingBalance<T, I>>
 	for FeeMarketPayment<T, I, Currency>
```

Comment:
There is a `RootAccount` in the darwina-common. `FeeMarketPayment<T, I, Currency, RootAccount>`.
what is the difference?

Reply:
Good catch. The main branch of `darwinia-bridges-substrate` repo includes such a change https://github.com/paritytech/parity-bridges-common/pull/962, and `darwinia-common` not yet. That makes the difference.

Reply:
thanks